aws/structure.go

```diff
@@ -5441,3 +5442,33 @@ func flattenRoute53ResolverRuleTargetIps(targetAddresses []*route53resolver.Targ
 
 	return vTargetIps
 }
+
+func flattenOrgRoots(roots []*organizations.Root) []map[string]interface{} {
```

Comment:
Could you please move this function and its unit testing into the resource (test) file since it is only used once? We are trying to move away from the catch-all `structure.go` where possible. Thanks!

Nit: We may also want these function names to use `Organizations` instead of `Org` just to remove any ambiguities in the future.

Reply:
All of your suggestions have been implemented.